page2img.py

```diff
@@ -114,7 +116,7 @@ def cli(page, out_dir, level, image_format, text, font):
         # extract text if requested by user
         if text:
             unic = struct.find("./" + PC + "TextEquiv").find("./" + PC + "Unicode")
-            if unic is not None:
+            if unic is not None and unic.text is not None:
```

Comment:
Would it be better to `print` a warning if an empty text was found?

Reply:
No. It's better like this.